admin/src/config.js

```diff
@@ -0,0 +1,15 @@
+const envs = {
+  development: {
+    API_URL: 'http://localhost:8082'
```

Comment:
we can't really config the admin API like this, it needs to switch based on current URL. See how the frontend determines the backend url